datahub-web-react/src/app/sharedV2/queryBuilder/GroupHeader.tsx

```diff
@@ -94,27 +71,25 @@ const GroupHeader = ({
             <ActionsContainer>
                 <ButtonComponent
                     variant="text"
-                    onClick={handleAddPropertyPredicate}
+                    onClick={onAddPropertyPredicate}
                     data-testid="query-builder-add-condition-button"
                 >
                     Add Condition
                 </ButtonComponent>
                 <ButtonComponent
                     variant="text"
-                    onClick={handleAddLogicalPredicate}
+                    onClick={onAddLogicalPredicate}
                     data-testid="query-builder-add-group-button"
                 >
                     Add Group
                 </ButtonComponent>
                 <CardIcons>
-                    {showDeleteButton && (
-                        <Icon
-                            icon="Delete"
-                            size="md"
-                            onClick={() => onDeletePredicate(index)}
-                            data-testid="query-builder-delete-button"
-                        />
-                    )}
```

Comment:
i'm a little confused by some of these changes - didn't we recently add the `showDeleteButton` logic? and above, why did we remove the e.preventDefault? didn't we add that to fix an issue we were seeing with the default form validation showing up for the name field for the hierarchy module modal?